apiary.apib

```diff
@@ -268,6 +268,45 @@ Any response that does not provide a ```200 OK``` response code should be treate
     + Attributes (Base Collection Response)
         + `data` (array[Business Asset Attribute Type], required)
 
+## Business Asset Type Group Collection [/businessassettypegroups{?page,pagesize,orderby,includeAudit}]
```

Comment:
Only suggestion is to add comments to these new endpoints so there is less to cleanup/add later  =)